Assets/GothicVR/Scripts/Caches/AssetCache.cs

```diff
@@ -161,7 +161,17 @@ public PxSoundData<float> TryGetSound(string key)
             if (soundCache.TryGetValue(preparedKey, out PxSoundData<float> data))
                 return data;
             
-            var wavFile = PxSound.GetSoundArrayFromVDF<float>(PhoenixBridge.VdfsPtr, $"{GetPreparedKey(key)}.wav");
+            PxSoundData<float> wavFile = new PxSoundData<float>();
```

Comment:
It's not needed to initialize the variable. This is sufficient:

```suggestion
            PxSoundData<float> wavFile;
```